doc/release/README.tasks

```diff
@@ -57,15 +57,14 @@ Chapel's default tasking layer implementation for most targets is based
 on the Qthreads user-level threading package from Sandia National Labs.
 This provides a lightweight implementation of Chapel tasking and will
 also ultimately provide an optimized implementation of sync variables.
-Qthreads tasking is the default for all targets except Intel KNC and
-Cygwin.  To use qthreads tasking, please take the following steps:
+To use qthreads tasking, please take the following steps:
 
 1) Ensure that the environment variable CHPL_HOME points to the
    top-level Chapel directory, as always.
 
 2) Set up your environment to use Qthreads:
 
-      ensure CHPL_TASKS is not set
+      ensure CHPL_TASKS is not set (if qthreads is the default)
```

Comment:
Could instead remove this and the "-- or --", and put "(only needed if qthreads is not the default)" on the "export CHPL_TASKS=qthreads" line.